### DISCUSSION TITLE
心动日剧 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#173] [模板讨论]
心动日剧 模板问题反馈评论区
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 1/3 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Response Error : HTTP 404: Not Found,
Request URL: http://www.doki8.net/wp-login.php

好像前天还挺好的，现在就不行了。

**Reply by wjf0214** [reply]
登录增加了验证码，所以失效了。有空我看下。

**Reply by wjf0214** [reply]
已经更新了。可以试试。

**Comment by miaoxiaozhao** [comment]
好像又失效了，不知道为啥。问题提示是一样的。

**Reply by wjf0214** [reply]
我抽空看看怎么做吧。现在改成了算术题，框架里实现起来不方便。

**Reply by wjf0214** [reply]
更新了，你可以试试。不过要用最新版框架。要框架内强更下。才可以用

**Reply by miaoxiaozhao** [reply]
我重装了QD，用最新的模板，还是不行

请老大再看看。多谢。

**Reply by miaoxiaozhao** [reply]
2023-06-26 17:38:00
日志：Failed at 20/22 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Request URL: http://www.doki8.net/login

错误日志。

**Reply by wjf0214** [reply]
需要qd强制更新，执行 `sh -f update.sh`，框架今天刚加的功能。

**Comment by miaoxiaozhao** [comment]
2023-09-13 16:21:54
日志：Failed at 20/22 request,
Fail assert: {"re": "302", "from": "status"} from success_asserts,
Request URL: http://www.doki8.net/login

是不是乘法的问题？

**Reply by wjf0214** [reply]
再试试
